sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -363,21 +363,22 @@ export interface ServiceBusClientOptions {
 
 // @public
 export interface ServiceBusMessage {
+    applicationProperties?: {
```

Comment:
@mohsin-mehmood 
"properties" under CorrelationRuleFilter is meant to match the "applicationProperties" on the message. 
Please update the "properties" under CorrelationRuleFilter to "applicationProperties".

Reply:
I have also renamed the `label` property to `subject` to match the ServiceMessage property